samples/OpenIdConnectSample/Startup.cs

```diff
@@ -39,10 +37,8 @@ public void ConfigureServices(IServiceCollection services)
                 sharedOptions.SignInScheme = CookieAuthenticationDefaults.AuthenticationScheme);
         }
 
-        public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
```

Comment:
We try to keep these in sync with the templates for things unrelated to the current feature:
https://github.com/aspnet/Templates/blob/dev/src/BaseTemplates/StarterWeb/Startup.cs